### PR TITLE
Add single node e2e test scenario

### PR DESCRIPTION
**Pull request by lut777**

### CODE REVIEW

**Comment by oilbeater** [comment]
Please squash to one commit

File: actionTrigger.md [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (oilbeater) [inline actionTrigger.md:1]: no need here

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Comment by lut777** [comment]
done.

**Comment by lut777** [comment]
done

File: actionTrigger.md (hunk 1 above)
Comment (lut777) [reply actionTrigger.md]: file removed.

**Review by oilbeater — APPROVED** [review]

**Pull request merged**